Makefile

```diff
@@ -168,6 +168,12 @@ runlocal-rp: ci-rp podman-secrets
 		--secret proxy.crt,target=/app/secrets/proxy.crt \
 		$(LOCAL_ARO_RP_IMAGE) rp
 
+.PHONY: runlocal-rp-the-old-way
+runlocal-rp-the-old-way:
+	echo -e "\e[31mIf you're running this instead of runlocal-rp because that's broken, there's probably a problem blocking the whole team. Has a Slack thread been started, and an owner assigned to fix it? Could it be you?\e[0m"
```

Comment:
ack - might be a `bash` thing... which this works for too! pushed.

Reply:
+1. The `echo` doesn't work as expected because us MacOS users have to install the GNU `coreutils` to replace the versions that come with MacOS, and the built-in `echo` on MacOS doesn't support `-e`.